armi/reactor/assemblyLists.py

```diff
@@ -25,35 +36,136 @@
 from armi.reactor.flags import Flags
 
 
+class AutoFiller(abc.ABC):
+    """
+    Class for governing automatic insertion of Assemblies.
```

Comment:
be more specific: insertion of assemblies without specifying exactly which location of a container they go in.